kasa/experimental/enabled.py

```diff
@@ -4,7 +4,7 @@
 class Enabled:
     """Class for enabling experimental functionality."""
 
-    value = False
+    value = True
```

Comment:
This can go back to false before merging but for now it's set to true so this can be user tested in HA.